ui/src/layouts/BuildPublication.tsx

```diff
@@ -60,9 +60,29 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
                     message: 'You have selected there is a conflict of interest, please provide a reason.'
                 };
             }
-            if (typeof store.conflictOfInterestStatus == 'undefined') {
+
+            if (typeof store.conflictOfInterestStatus === 'undefined') {
```

Comment:
Looking at the requirements, this option should never be undefined. It's 'true' by default, that means the only other option is 'false'. But never undefined

Reply:
This is not in the scope of the ticket, this is to do with conflict of interest, I just updated the equality to check with an extra '='